media/src/Chapter.svelte

```diff
@@ -67,32 +67,49 @@
             curveId = null;
         } else {
             const params = exampleCurveParams[selection];
-            const [X,Y,Z,A,B] = ["x","y","z","a","b"].map((c) => math.parse(params[c]));
+            const [X,Y,Z,A,B] = ["x","y","z","a","b"].map((c) => 
+                                    math.parse(params[c]));
             curveId = uuidv4();
 
             boxes = [...boxes, {id: curveId, kind: "curve", params, }];
-            texStrings.r = "\\left \\langle " + X.toTex() + ", " + Y.toTex()  + ", " + Z.toTex() + "\\right \\rangle";
+            texStrings.r = "\\left \\langle " + X.toTex() + ", " + Y.toTex() +
+                           ", " + Z.toTex() + "\\right \\rangle";
             texStrings.a = A.toTex();
             texStrings.b = B.toTex();
         }
     }
 
     </script>
 
-<h1 class="top-title" on:click={toggleHidden}>Arc Length &amp; Curvature</h1>
+<h1 class="top-title" on:click={toggleHidden} on:keyup={toggleHidden}>
+    Arc Length &amp; Curvature
+</h1>
 <article class:hidden>
 
-    <p>Suppose we have a curve <M>C</M> in space parameterized by a smooth function <M>\mathbf r(t)</M> for <M>a \leq t \leq b</M> and we wish to know how long it is. That is, we want to compute the <b>arc length</b> of <M>C</M>.</p>
+    <p>
+        Suppose we have a curve <M>C</M> in space parameterized by a smooth 
+        function <M>\mathbf r(t)</M> for <M>a \leq t \leq b</M> and we wish 
+        to know how long it is. That is, we want to compute the 
+        <b>arc length</b> of <M>C</M>.
+    </p>
 
-    <p>Select an example: <span on:click={() => addCurve(0)}>crown</span> | <span on:click={() => addCurve(1)}>twist</span> </p>
+    <p>Select an example: 
+        <button on:click={() => addCurve(0)} on:keyup={addCurve(0)}>
```

Comment:
After seeing these changes I think I realized our confusion about why this is required in the first place: a `<button>` with an on:click handler is already considered accessible by svelte - as we were seeing, it's already keyboard-accessible by nature of it being a button.

eslint was only complaining about the non-button elements like `<span>` that have handlers. There are many `<button>` elements in App.svelte with on:click handlers, though this branch is still passing eslint.

So, long story short, you can even remove the on:keyup handlers for these new button elements you've made here.

Reply:
Converted the elements to buttons, but I'll reserve the styling and adding classes to another PR